setup.py

```diff
@@ -56,6 +61,32 @@ def get_extensions():
     return ext_modules
 
 
+class build_custom_ops(distutils.command.build.build):
+    def run(self):
+        build_dir = os.path.join('build', 'custom_ops')
+        inst_dir = os.path.join(self.build_lib, 'maskrcnn_benchmark', 'lib')
+        os.makedirs(build_dir, exist_ok=True)
```

Comment:
This is unfortunately Python3-only. :-/
There is an implementation of `makedirs` in `maskrcnn-benchmark/utils/misc` that implements a Python2-compatible version of `exist_ok`. Maybe you could copy it to this file and use it?

Reply:
Thanks!